docs/security/trust/trust_delegation.md

```diff
@@ -0,0 +1,199 @@
+<!--[metadata]>
++++
+title = "Delegations for content trust"
+description = "Delegations for content trust"
+keywords = ["trust, security, delegations, keys, repository"]
+[menu.main]
+parent= "smn_content_trust"
++++
+<![end-metadata]-->
+
+# Delegations for content trust
+
+Docker Engine 1.10 and above supports the usage of the `targets/releases`
+delegation as the canonical source of a trusted image tag, if it exists.
+
+Using this delegation allows you to collaborate with other publishers without
+sharing your repository key (a combination of your targets and snapshot keys -
+please see "[Manage keys for content trust](trust_key_mng.md)" for more information).
+A collaborator can can keep their own delegation key private.
+
+Delegations are currently an optional feature - in order to set up delegations,
+you must use the Notary CLI: [download it](https://github.com/docker/notary/releases) and ensure that it is
+available on your path.
+
+For more detailed information about how to use Notary outside of the default
+Docker Content Trust use cases, please read the
+[the Notary CLI documentation](https://github.com/docker/notary/blob/master/docs/getting_started.md).
+
+Note that when publishing and listing delegation changes using the Notary client,
+your Docker Hub credentials are required.
+
+## Generating delegation keys
+
+Your collaborator needs to generate a private key (either RSA or ECDSA)
+and give you the public key so that you can add it to the `targets/releases`
+delegation.
+
+The easiest way to for them to generate these keys is with OpenSSL.
+Here is an example of how to generate a 2048-bit RSA portion key (all RSA keys
+must be at least 2048 bits):
+
+```
+$ opensl genrsa -out delegation.key 2048
+Generating RSA private key, 2048 bit long modulus
+....................................................+++
+............+++
+e is 65537 (0x10001)
+
+```
+
+They should keep `delegation.key` private - this is what they will use to sign
+tags.
+
+Then they need to generate a x509 certificate containing the public key, which is
+what they will give to you.  Here is the command to generate a CSR (certificate
+signing request):
+
+```
+$ openssl req -new -sha256 -key delegation.key -out delegation.csr
+```
+
+Then they can send it to whichever CA you trust to sign certificates, or they
+can self-sign the certificate (in this example, creating a certificate that is
+valid for 1 year):
+
+```
+$ openssl x509 -req -days 365 -in delegation.csr -signkey delegation.key -out delegation.crt
```

Comment:
we should have openssl instructions, but should also have a notary command to generate all of this for the users.